content/docs/0.11.x/continuous_delivery/deployment_helm/index.md

```diff
@@ -5,18 +5,18 @@ weight: 10
 keywords: [0.11.x-cd]
 ---
 
-Keptn uses [Helm v3](https://helm.sh/) for deploying a services to a Kubernetes cluster. This is currently implemented in the [helm-service](https://github.com/keptn/keptn/tree/0.11.0/helm-service).
+Keptn uses [Helm v3](https://helm.sh/) for deploying a services to a Kubernetes cluster. This is currently implemented in the [helm-service](https://github.com/keptn/keptn/tree/0.11.2/helm-service).
 Keptn's helm-service supports the following deployment strategies:
 
 * **Direct deployments**
 * **Blue-green deployments**
 * **user-managed deployments** (experimental)
 
-The explanation below is based on the provided Helm Chart for the carts microservice, see [Charts](https://github.com/keptn/examples/tree/0.11.0/onboarding-carts/carts) for details.
+The explanation below is based on the provided Helm Chart for the carts microservice, see [Charts](https://github.com/keptn/examples/tree/0.10.0/onboarding-carts/carts) for details.
```

Comment:
Can we please release: https://github.com/keptn/examples with version 0.11. to keep it aligned with the Keptn releases?